packages/fxa-content-server/app/scripts/views/survey.js

```diff
@@ -0,0 +1,115 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/*eslint-disable no-unused-vars*/
+import React, { useState, useEffect } from 'react';
+/*eslint-enable no-unused-vars*/
+import ReactDOM from 'react-dom';
+import Cocktail from 'cocktail';
+import BaseView from './base';
+import SettingsPanelMixin from './mixins/settings-panel-mixin';
+
+/*eslint-disable no-unused-vars*/
+import Survey, { CreateHandleIframeTask } from '@fxa-components/Survey/index.tsx';
+/*eslint-enable no-unused-vars*/
+
+/*
+config = {
+  surveys: [
+    {
+      id: 'portugese-speaking-mobile-users-in-southern-hemisphere',
+      conditions: [{platform: 'mobile', region: 'southernHemisphere', lang: 'pt', relier: 'email'}],
+      view: ['settings-home'],
+      rate: 0.1,
+      url: 'https://www.surveygizmo.com/s3/5541940/portugese-speaking-mobile-users-in-southern-hemisphere'
+    }
+  ]
+}
+
+There will be multiple surveys at times.
+
+Will need to set a date in local or session storage to indicate
+the last time a survey was shown to the user. We want to
+avoid showing the user more than one survey per month.
+
+*/
+
+/*eslint-disable no-unused-vars*/
+const SurveyView = surveyURL => {
+  /*eslint-enable no-unused-vars*/
+  const [showSurvey, setShowSurvey] = useState(true);
+  const [surveyComplete, setSurveyComplete] = useState(false);
+
+  useEffect(() => {
+    const handleIframeTask = CreateHandleIframeTask(() => {
+      setSurveyComplete(true);
+      setTimeout(() => {
+        setShowSurvey(false);
+      }, 300);
+    });
```

Comment:
Are these always updated together?  That is, can `showSurvey` and `surveyComplete` both be true or both false?  If not, we can use a single state here I think.

Reply:
They are separate so that we can wait about 3 seconds before hiding the survey to give the user a chance to see the surveyComplete message